### PR TITLE
Added Layer5 in the news section

### LINKED ISSUE #1563: [NG] Implementation of Layer5 in the news section on homepage
**Description**
Currently Layer5 in the news section is not implemented.

**Desired Situation**

 This section should be completed and appear similar in style to [this mockup](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?node-id=3359%3A0)

---
[Optional] **Screenshots**

Screen Shot 2021-02-10 at 10 15 50 PM
[Optional] **Additional context**

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by chinmaym07**
Signed-off-by: chinmaym07 <b418020@iiit-bh.ac.in>

**Description**

This PR fixes #1563 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Home/So-Special-Section/so-special-style.js [hunk 1]
```diff
@@ -94,11 +99,23 @@ const SoSpecialWrapper = styled.div`
         color:black;
     }
 
-    
+    .so-special-foot{
+        text-align:center;
+        font-family:"Open Sans";
```
Comment (Nikhil-Ladha) [inline src/sections/Home/So-Special-Section/so-special-style.js:104]: Not needed

File: src/sections/Home/So-Special-Section/so-special-style.js [hunk 2]
```diff
@@ -7,6 +7,8 @@ const SoSpecialWrapper = styled.div`
     align-items:center;
     padding:0rem 3rem 2rem 3rem;
     overflow:hidden;
+    font-family:"Qanelas Soft";
```
Comment (Nikhil-Ladha) [inline src/sections/Home/So-Special-Section/so-special-style.js:10]: Fonts shouldn't be used this way. Qanelas soft is specifically for headings, if you want to use this font use the `<h*>` tags

File: src/sections/Home/So-Special-Section/so-special-style.js [hunk 3]
```diff
@@ -7,6 +7,8 @@ const SoSpecialWrapper = styled.div`
...
+    font-weight:600;
```
Comment (Nikhil-Ladha) [inline src/sections/Home/So-Special-Section/so-special-style.js:11]: Should be updated, while updating the above.

File: src/sections/Home/So-Special-Section/index.js [hunk 4]
```diff
@@ -89,7 +89,8 @@ const SoSpecial = () => {
   return(
     <SoSpecialWrapper>
       <div className="so-special-head">
-        <h1>Why Is Layer5 So special?</h1>
+        <h4>LAYER5 IN THE NEWS</h4>
```
Comment (Nikhil-Ladha) [inline src/sections/Home/So-Special-Section/index.js:92]: @leecalcote should this be capitalized?

File: src/sections/Home/So-Special-Section/so-special-style.js (hunk 3 above)
Comment (chinmaym07) [reply src/sections/Home/So-Special-Section/so-special-style.js:11]: Thanks for the review . I 'll make the required changes !

File: src/sections/Home/So-Special-Section/so-special-style.js [hunk 5]
```diff
@@ -93,70 +93,59 @@ const SoSpecialWrapper = styled.div`
...
     }
-
-    
+    .so-special-foot{
+        text-align:center;
+        .so-special-foot-btn
+        {
+            color:white;
+            margin:2rem;
+            :hover{
```
Comment (Nikhil-Ladha) [inline src/sections/Home/So-Special-Section/so-special-style.js:102]: Hover effect is set for the button component

Comment (chinmaym07) [reply src/sections/Home/So-Special-Section/so-special-style.js:102]: Yes I agree . But I think color of the text inside it should change to black on hover . If I remove it ,text color is not changing ,that's why I have set it up separately.

**Review by Nikhil-Ladha — APPROVED** [review]
Lgtm

**Pull request merged**